project/plugins.sbt

```diff
@@ -5,6 +5,6 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.4")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```

Comment:
This is unrelated, but while we're in here can we get rid of the code coverage stuff? I find it more annoying than helpful.

Reply:
Also from what I can tell the coverage is what's making CI fail right now.

Reply:
decided to hold off on removal for now.